keyserver/listener.go

```diff
@@ -32,7 +33,6 @@ func (server *ConiksServer) listenForRequests(ln *net.TCPListener, handler func(
 			log.Printf("accept client: %s", err)
 			continue
 		}
-		conn = tls.Server(conn, server.tlsConfig)
```

Comment:
We could ... I did this to match the client (and also because I was called `c.CloseWrite()` after the write here, but that wasn't necessary since we're closing the entire connection).

But, out of curiosity, why do you prefer that?